src/libs/ReportUtils.ts

```diff
@@ -4317,6 +4317,29 @@ function isGroupChat(report: OnyxEntry<Report>): boolean {
     );
 }
 
+/**
+ * Assume any report without a reportID is unusable.
+ */
+function isValidReport(report?: OnyxEntry<Report>): boolean {
+    return Boolean(report?.reportID);
+}
+
+/**
+ * Check to see if we are a participant of this report.
+ */
+function isReportParticipant(accountID: number, report?: OnyxEntry<Report>): boolean {
+    if (!accountID) {
+        return false;
+    }
+
+    // We are not the owner or the manager or a participant
+    if (report?.ownerAccountID !== accountID && report?.managerID !== accountID && !(report?.participantAccountIDs ?? []).includes(accountID)) {
```

Comment:
BE returns ownerAccountID as '0' and this check is failing for report owner
<img width="1437" alt="Screenshot 2024-01-23 at 12 03 53 PM" src="https://github.com/Expensify/App/assets/102477862/6996bb38-e11e-45cd-a6fa-aff545bb9ed0">

Reply:
Oh I see. In the case of a DM - you would not be in the `participantAccountIDs`?

Reply:
DMs are supposed to have an `ownerAccountID` of `0` that is expected.

Reply:
But the problem is that they won't be in there... have a solution one sec.

Reply:
Let me know if the latest commit looks better!

Reply:
Yes, this gets correct valid report length,